mw-webapp/src/service/auth/logOut.ts

```diff
@@ -1,9 +1,11 @@
 import {signOut} from "firebase/auth";
 import {auth} from "src/firebase";
+import {USER_IS_AUTH, USER_UNREGISTERED} from "src/service/auth/constants";
 
 /**
  * Call method for logout
  */
 export const logOut = async () => {
   await signOut(auth);
+  localStorage.setItem(USER_IS_AUTH, USER_UNREGISTERED);
```

Comment:
why if user press logout he bacame unregistered? What does registration means?

Reply:
Yes, there is a logout going on here.